<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCallHistory</class>
 <widget class="QWidget" name="FCallHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../Resort/res.qrc">
       <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deStart">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deEnd">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_2">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../Resort/res.qrc">
       <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQCheckBox" name="chIn">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Incoming</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQCheckBox" name="chOut">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Outgoing</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQCheckBox" name="chInt">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Internal</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQCheckBox" name="chTOut">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Transfer out</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQCheckBox" name="chTIn">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Transfer in</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>324</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">eqcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
